README.md

```diff
@@ -1,6 +1,9 @@
 # RLG
-RandomLifeGenerator
 
-Project originally created during classes with some of my friends, that I upgraded and continued even after. It's such really basic, works with Python. As say in the title, it allows you to create random and totally WTF lifes for the personns you want. Just gotta change the names in the lists which are meant for this.
+> RandomLifeGenerator
+
+This project has originally been created during classes with some of my friends. I then continued working on it. It's really basic, and works with just Python, no dependencies.
```

Comment:
We would like to see the source code one day :-)

Also, python exists in many different versions. Do we need a specific python version to run it? What version of python is it tested on?